cmd/operator-sdk/test/local.go

```diff
@@ -220,10 +240,24 @@ func testLocalGoFunc(cmd *cobra.Command, args []string) error {
 	if tlConfig.goTestFlags != "" {
 		testArgs = append(testArgs, strings.Split(tlConfig.goTestFlags, " ")...)
 	}
-	if tlConfig.namespace != "" || tlConfig.noSetup {
-		testArgs = append(testArgs, "-"+test.SingleNamespaceFlag, "-parallel=1")
+	if tlConfig.operatorNamespace != "" || tlConfig.noSetup {
+		testArgs = append(testArgs, "-parallel=1")
 	}
-	env := append(os.Environ(), fmt.Sprintf("%v=%v", test.TestNamespaceEnv, tlConfig.namespace))
+	env := os.Environ()
+	if tlConfig.operatorNamespace != "" {
+		env = append(
```

Comment:
WDYT about add a log to inform that the it will be usin the operatorNamespace == value?

Reply:
we cannot be sure about it here.
if the `operator-namespace` is not set explicitly, the a operaotor-namespace is created by
`ctx.GetOperatorNamespace()` (or in the deprecated `ctx.GEtNamespace()`).

It is created at random
`ctx.namespace = ctx.GetID()`

so a log here can be confusing.